modules/order/order.service.js

```diff
@@ -86,19 +86,19 @@ class OrdersService {
     throw new Error(ORDER_NOT_FOUND);
   }
 
-  async updateOrder(id, order) {
-    if (!ObjectId.isValid(id)) {
+  async updateOrder(order) {
+    if (!ObjectId.isValid(order._id)) {
       throw new Error(ORDER_NOT_VALID);
     }
-    const orderToUpdate = await Order.findById(id);
+    const orderToUpdate = await Order.findById(order._id);
     if (!orderToUpdate) {
       throw new Error(ORDER_NOT_FOUND);
     }
 
     if (order.items || order.delivery || order.address) {
       const totalItemsPrice = this.calculateTotalItemsPrice(order.items);
 
-      if (order.delivery.sentBy === 'Nova Poshta') {
+      if (orderToUpdate.delivery.sentBy!=='Nova Poshta'&&order.delivery.sentBy === 'Nova Poshta') {
```

Comment:
It seems you did not set up prettier because you do not have spaces between `&&`. Please, set up it and let it go through your code, to make it beautiful.

Reply:
fixed